service/common.go

```diff
@@ -157,3 +157,13 @@ func DefaultEnvs(backend *backupv1alpha1.Backend, config config.Global) []corev1
 	}...)
 	return vars
 }
+
+// BuildTagArgs will prepend "--tag " to every element in the given []string
+// TODO: this is wrong, needs fixing!!
```

Comment:
What's wrong about it?

Reply:
That can be removed, I fixed the underlying issue, but forgot to remove the `TODO:`